server/src/main/java/org/opensearch/index/translog/TranslogWriter.java

```diff
@@ -110,6 +110,8 @@ public class TranslogWriter extends BaseTranslogReader implements Closeable {
 
     private final Map<Long, Tuple<BytesReference, Exception>> seenSequenceNumbers;
 
+    private final Boolean remoteTranslogEnabled;
```

Comment:
Lets not pollute this with `remoteTranslogEnabled ` maybe just use `fsync` instead?

Reply:
Fsync is a property of file channel and remote or not is specific to translog. Putting fsync here will be out of its context.